Zend/zend_compile.c

```diff
@@ -5501,17 +5498,30 @@ void zend_compile_closure_uses(zend_ast *ast) /* {{{ */
 }
 /* }}} */
 
+static void zend_check_magic_method_attr(uint32_t attr, const char* entity) /* {{{ */
+{
+	if (memcmp(entity, "__callStatic", 12) == 0) {
```

Comment:
To avoid counting mistakes, it is better to replace the `12` with `sizeof("__callStatic") - 1`.